irispy/iris_tools.py

```diff
@@ -82,8 +82,8 @@ def get_iris_response(time_obs=None, pre_launch=False, response_file=None, respo
 
     Parameters
     ----------
-    time_obs: a `numpy.array` of floats, as a kwarg, valid for version > 2
-        Observation times of the datapoints. Must be in the format of the output of time_obs = parse_time('2013-09-03', format='utime'), which yields 1094169600.0 seconds. The argument time_obs is ignored for versions 1 and 2.
+    time_obs: an `astropy.time.Time` object, as a kwarg, valid for version > 2
+        Observation times of the datapoints. Must be in the format of , e.g., time_obs = parse_time('2013-09-03', format='utime'), which yields 1094169600.0 seconds in value. The argument time_obs is ignored for versions 1 and 2.
```

Comment:
PEP 8 compliance: Can you break this into multiple lines no line hits the 100 character mark?